bin/diagnostic.js

```diff
@@ -1 +1,12 @@
 'use strict';
+const ask = require('../lib/ask.js');
+
+let answer = 'x;
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (x > 1 && < 10) {
+   console.log("Guess again, and choose between 1 and 10");
+   break;
+  }
+}
```

Comment:
Please study this example solution below and make sure you understand what is happening in this code.
```
const ask = require('../lib/ask.js');

let answer = '';

while (answer !== '5') {
  if (answer < 1 || answer > 10) {
   console.log('Please try a number between 1 and 10');
 } else if (answer !== '') {
   console.log('That\'s not it!');
 }
  answer = ask("Guess my secret? ");
}

console.log("You guessed it!");
```